instrumentor/controllers/sourceinstrumentation/common.go

```diff
@@ -98,97 +98,91 @@ func syncWorkload(ctx context.Context, k8sClient client.Client, scheme *runtime.
 		return ctrl.Result{}, err
 	}
 
-	podWorkload := k8sconsts.PodWorkload{
-		Name:      obj.GetName(),
-		Namespace: obj.GetNamespace(),
-		Kind:      workload.WorkloadKindFromClientObject(obj),
```

Comment:
reviewer note: this is the only placed where we used this function, so now it can be removed from odigos